Makefile

```
@@ -87,6 +87,10 @@ KUBEADM_BOOTSTRAP_CONTROLLER_IMG ?= $(REGISTRY)/$(KUBEADM_BOOTSTRAP_IMAGE_NAME)
 KUBEADM_CONTROL_PLANE_IMAGE_NAME ?= kubeadm-control-plane-controller
 KUBEADM_CONTROL_PLANE_CONTROLLER_IMG ?= $(REGISTRY)/$(KUBEADM_CONTROL_PLANE_IMAGE_NAME)
 
+# operator
+OPERATOR_IMAGE_NAME ?= cluster-api-operator-controller
```

```suggestion
OPERATOR_IMAGE_NAME ?= cluster-api-operator
```
What about dropping controller?